### PR TITLE
Added a generation for values.yaml for a seed cluster

### PR DESCRIPTION
**Pull request by glower**
**What this PR does / why we need it**:
For an easy installation on a seed cluster, we need an example `values.yaml` we can refer from the documentation.  The new `values.seed.example.yaml` contains only `kubermatic/values.yaml`

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by mrIncompetent** [comment]
Assigning @xrstf as its charts related

File: api/hack/sync-charts.sh [hunk 1]
```diff
@@ -34,10 +35,17 @@ git checkout ${INSTALLER_BRANCH}
...
 
 rm -f ${TARGET_DIR}/values.yaml
-rm -f ${TARGET_VALUES_FILE}
-touch ${TARGET_VALUES_FILE}
 
-echo "# THIS FILE IS GENERATED BY https://github.com/kubermatic/kubermatic/blob/master/api/hack/sync-charts.sh" >> ${TARGET_VALUES_FILE}
+VALUES=(${TARGET_VALUES_FILE} ${TARGET_VALUES_SEED_FILE})
+for VALUE_FILE in "${VALUES[@]}"; do
+  rm -f ${VALUE_FILE}
+  touch ${VALUE_FILE}
```
Comment (thz) [inline api/hack/sync-charts.sh]: remove the `touch` and use `>` instead of `>>`.

File: api/hack/sync-charts.sh [hunk 2]
```diff
@@ -34,10 +35,17 @@ git checkout ${INSTALLER_BRANCH}
 cd ..
 
 rm -f ${TARGET_DIR}/values.yaml
-rm -f ${TARGET_VALUES_FILE}
-touch ${TARGET_VALUES_FILE}
 
-echo "# THIS FILE IS GENERATED BY https://github.com/kubermatic/kubermatic/blob/master/api/hack/sync-charts.sh" >> ${TARGET_VALUES_FILE}
+VALUES=(${TARGET_VALUES_FILE} ${TARGET_VALUES_SEED_FILE})
+for VALUE_FILE in "${VALUES[@]}"; do
```
Comment (thz) [inline api/hack/sync-charts.sh]: how about skipping the VALUES variable and just do a:

```
for VALUE_FILE in ${TARGET_VALUES_FILE} ${TARGET_VALUES_SEED_FILE}; do
...
```
When in doubt - go with the fewer lines of code...

Comment (glower) [reply api/hack/sync-charts.sh]: done

File: api/hack/sync-charts.sh (hunk 1 above)
Comment (glower) [reply api/hack/sync-charts.sh]: done

**Comment by thz** [comment]
(i did not review the actual content but only shell style)

**Comment by glower** [comment]
@mrIncompetent can you have a look if the created values file is enough for the seed cluster?

**Comment by mrIncompetent** [comment]
https://github.com/kubermatic/kubermatic/pull/2028#issuecomment-422727397

**Review by xrstf — APPROVED** [review]
LGTM

**Pull request merged**